code/datalab-app/src/app/components/welcome/DescribeDatalabs.js

```diff
@@ -3,6 +3,7 @@ import DescribeElement from './DescribeElement';
 import slackLogo from '../../../assets/images/slack-hori.png';
 import gitHubLogo from '../../../assets/images/github-hori.png';
 import discourseLogo from '../../../assets/images/discourse-hoir.png';
```

Comment:
Not in this PR, but is this supposed to be `discourse-hori.png`?

Reply:
Yes it is.